packages/typespec-azure-portal-core/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../core/packages/tsconfig.json",
+  "composite": true,
+  "references": [
+    { "path": "../../core/packages/compiler/tsconfig.json" },
+    { "path": "../../core/packages/rest/tsconfig.json" },
+    { "path": "../../core/packages/openapi/tsconfig.json" },
+    { "path": "../../core/packages/lint/tsconfig.json" }
```

Comment:
Shouldn't need that one

Reply:
deleted it :) thank you!

Reply:
sorry I just meant the last one(lint), you technically want this to be a list of all the local dependencies you have if you want to be able to do `rush watch` and work on thoses pacakges so it knows in which order to build. It probably doesn't matter for this one as its unlikely you'd be working on those core packages too but this is good to keep in sync with dependencies.

Reply:
got it! yes, reverted back and deleted only last one :) thank you!